src/challenger/op-challenger/op_challenger_launcher.star

```diff
@@ -112,6 +112,12 @@ def get_challenger_config(
         "--trace-type=" + ",".join(challenger_params.cannon_trace_types),
     ]
 
+    # TraceTypeSupper{Cannon|Permissioned} needs --cannon-depset-config to be set
+    # Added at https://github.com/ethereum-optimism/optimism/pull/14666
+    # Temporary fix: Add a dummy flag
+    # Tracked at issue https://github.com/ethpandaops/optimism-package/issues/189
+    cmd.append("--cannon-depset-config=dummy-file.json")
+
     # configure files
 
```

Comment:
If op-deployer is gonna generate that file, we could generate the right command already I think

```
        "--cannon-depset-config={0}/depsets.json".format(
            ethereum_package_constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS,
        ),
```

Also presumably that flag is required only if interop is enabled?

Reply:
This flag is required when cannon_trace_types contains super types which monorepo interop.yaml has it: https://github.com/ethereum-optimism/optimism/blob/6f68dc35e103278e366d2b8ba178ca87bbaacb0c/kurtosis-devnet/interop.yaml#L88

> Also presumably that flag is required only if interop is enabled?

I think yes, but deferring this question to @ajsutton 

I tested that on my local kt-devnet that `--cannon-depset-config` is only required when cannon_trace_types contains super types.

Note that this is a temporal/dummy fix. Made a follow up issue which I mentioned upper.

So ideally, we finally need the logic like `if game type is super, add that new flag`.

Reply:
ok, so I think we should guard this with

```
if interop_params.enabled:
  ...
```

That same flag is/will be used to tell op-deployer to
- generate depsets.json
- deploy the right dispute game

Reply:
Yes this is only required when enabling super-cannon game type (for interop).  It should be pointed to the same dependency set config file that is supplied to op-supervisor.